account_move_contact_attribute/readme/DESCRIPTION.md

```diff
@@ -1,4 +1,9 @@
 This module adds some contact-related fields to account.move.
 
-- Contact Person
-- Site Name
+- Secondary Partner: Records the name of a secondary partner 
+where the main partner supplies products.
+- Contact Person: Used to enter the partner's representative. 
+This is printed on the invoice.
+- Site Name: Used to specify a location, 
+useful for indicating detailed delivery destinations. 
+This is printed on the invoice.
```

Comment:
I feel like this should be added in `USAGE.md`.
We should also add this for `sale_order_contact_attribute`.

Reply:
I think we can keep this part as is.